pages/ForgetPassword/index.tsx

```diff
@@ -0,0 +1,204 @@
+import React, {useState} from 'react';
+import { RouteComponentProps } from 'react-router-native';
+import { Dispatch } from 'redux';
+import { View, ViewStyle, StyleSheet, TextStyle, TextInput, ScrollView, TouchableOpacity, Image, ImageStyle, ImageBackground, Platform } from 'react-native';
+import { AppConstants, AppTheme } from '../../config/DefaultConfig';
+import ThemedText from '../../components/UI/ThemedText';
+import useConstants from '../../hooks/useConstants';
+import RoundButton from '../../components/Base/RoundButton';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+import useTheme from '../../hooks/useTheme';
+
+const isIOS = (): Boolean => Platform.OS == "ios";
+
+// @ts-ignore
+const ImagePath = require("../../images/dual-tone.png");
+const forget = require("../../images/forget.png");
+
+interface Props extends RouteComponentProps {
+  dispatch: Dispatch,
+  history: any
+}
+
+const ForgetPassword: React.FunctionComponent<Props> = ({
+  history
+}: Props) => {
+  const constants: AppConstants = useConstants();
+  const theme: AppTheme = useTheme();
+
+  const backButton = () => {
+    history.push('/')
+  }
+
+  return (
+    <View style={{flex: 1, flexDirection:'column'}}>
+      <ScrollView>
+        <ImageBackground source={ImagePath} style={{ width: '100%', height: 550,}} >
```

Comment:
I see a lot of inline style not sure why we can't put them below in the StyleSheet object.

Reply:
I have fixed this